src/utils/update/managing.officer.mapper.ts

```diff
@@ -0,0 +1,97 @@
+import { mapAddress, isSameAddress } from "./mapper.utils";
+import { CompanyOfficerResource, FormerNameResource } from "@companieshouse/api-sdk-node/dist/services/company-officers/types";
+import { ManagingOfficerIndividual } from "../../model/managing.officer.model";
+import { ManagingOfficerCorporate } from "../../model/managing.officer.corporate.model";
+import { InputDate, yesNoResponse } from "../../model/data.types.model";
+
+export const mapToManagingOfficer = (officer: CompanyOfficerResource): ManagingOfficerIndividual => {
+  console.log(officer);
+  const raw = officer as any;
+  const service_address = mapAddress(officer.address);
+  const address = mapAddress(officer.address);
+  const names = splitNames(officer.name);
+  const formernames = getFormerNames(officer.former_names);
+
+  const result: ManagingOfficerIndividual = {
+    id: raw.links?.self,
+    first_name: names[0],
+    last_name: names[1],
+    has_former_names: raw.former_names ? yesNoResponse.Yes : yesNoResponse.No,
+    former_names: formernames,
+    date_of_birth: mapDateOfBirth(officer),
+    nationality: officer.nationality,
+    // appointed_on: officer.appointed_on,
```

Comment:
where did you get officer.appointed_on? MOs don't need an appointed date I think